src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random random = new Random();
+    private final Color[] colors = Color.values();
+
     public String getRandomColor() {
-        return null;
+        int i = random.nextInt(colors.length);
+        return colors[i].toString();
```

Comment:
[make sure you go thru checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants)

Reply:
I don't understand, where is the problem

Reply:
You indicated on the problem "Use name() for getting String representation of enum constants"